### DISCUSSION TITLE
制度的优越性

### DISCUSSION THREAD

**Discussion by Allenliang1024** [WeNeedHome/SummaryOfLoanSuspension#390] [General]
通过最近的一系列问题，包括但不限于食品安全、村镇银行、停贷等，归根到底是怎样限制公权力。
大家还觉得社会主义制度有啥优越性吗？
社会主义制度的意识形态是：公平正义、人类解放、共产主义、人民当家做主。
资本主义制度的意识形态是：自由民主、三权分立。

**Comment by yjeff112** [comment]
利好《毛选》

**Reply by white-cat** [reply]
这里不要谈论制度的问题！

**Reply by zxcs001** [reply]
> 要谈论制度的问题！

为什么不要讨论，做奴隶的自我修养？

我看到国内这帮韭菜们，在私下里说个话，还一副小心翼翼的那种模样。发自内心的让人感到恶心……

**Reply by motion-dream** [reply]
> 

做公共品，还记得以前90年是分房的。

**Reply by zxcs001** [reply]
> 做公共品，还记得以前90年是分房的。

说分房的，估计你也就20岁出头吧。哪有那么多房给你分？你知道00年之前，城市人均居住面积是多少？你以为国家做慈善的啊？当年上山下乡，你以为真是让城里年轻人去农村学习去的？学习学个快十多年？

现在买不起当地城市核心区域最好房产的人，当年你也别想分房的。住房指标每年就那么些，大家都是八仙过海，论资排辈，最后那么寥寥几个人拿到的房子，还说不定是别人换下来的房子。

**Reply by yanwuguangjun** [reply]
> > 要谈论制度的问题！
> 
> 为什么不要讨论，做奴隶的自我修养？
> 
> 我看到国内这帮韭菜们，在私下里说个话，还一副小心翼翼的那种模样。发自内心的让人感到恶心……

真正私下 并没有小心翼翼的  但是 在很多地方我们确实是小心翼翼的，没办法没得选，我们也不想，但是即使是放开让你随便说 ，你也不能说脱离现实的话吧。

**Reply by TomMao-CN** [reply]
> 这里不要谈论制度的问题！

本质就是制度问题，为啥不让讨论

**Comment by kuro523** [comment]


**Reply by xiaohao587** [reply]
> 补充一点。党员现在有近一亿了，我们国家有这么多的省份这么多的乡镇，每一层的行政人员数不胜数。对于百姓来说，百姓可不管你是中央政府还是地方政府，统一叫做政府。但是从概率上也很容易知道，这么多党员要出几个腐败分子不要太容易，就是各位程序员平日里工作，遇到些个奇葩主管和hr也不要太容易。所以国家提的纯洁党的队伍是非常非常非常非常非常非常...非常重要的一件事，只能留下践行党的核心价值观的人，不断的剔除投机倒把分子。所以从这个角度上看，非常支持各位怒气满满的业主发声和往上倒逼

现在中国和社会主义没有一点关系，总得要选一个社会性质吧，那就选了一个社会主义，至于是不是社会主义答案很肯定的，即使当初搞资本主义，几十年后还是这个吊样子

**Reply by Joseph-Wheat-Carryier** [reply]
嗯嗯， 那穷人的核心问题在于限制自己变成富人？

**Reply by liuwenwu520** [reply]
公务员资产不透明，多数政务不公开，到处都充斥着猫腻，问题多的很

**Reply by kuro523** [reply]
刚刚看了下，有人说，“”我相信对根本制度问题的讨论，不是志愿者做这个活动的目的“”。很赞同，所以我把我的长篇大论都自觉删了。希望各位业主能有好的结局

**Reply by woshizengling** [reply]
> 刚刚看了下，有人说，“”我相信对根本制度问题的讨论，不是志愿者做这个活动的目的“”。很赞同，所以我把我的长篇大论都自觉删了。希望各位业主能有好的结局

还是有明白人，单纯讨论批判制度的人，我觉得无论什么样的制度，他们总能找到批判点

**Comment by xiejun946** [comment]
没有什么优势吧，不过资本主义也好不到哪里去，主要是看贫富差距哪种制度大，中国可以养活14亿人口充分说明制度的可行性

**Reply by wedxz** [reply]
好死不如赖活着.
毕竟能把这么多人养到不敢生孩子也是牛皮

**Reply by HunDeGong1664** [reply]
那么问题来了，是14亿人养了这个政府，还是这个政府养了14亿人

**Reply by volkerzht** [reply]
这就是被教育成功的典范，养活14亿人的是我们自己，我们劳动，我们交税，能来这的个税都少交不了，外加各种消费税。是我们养活了这些公务员，养活了这个政府，而不是他们养活我们，我们不买东西，不交税，他们瞬间就会急，就会用铁拳打人，如果这也叫养活的话，听我说，谢谢你。

**Reply by aliseming** [reply]
是14亿人养活了这么庞大的体制吧，还有党政两套，庞大的开销，维稳费，向外国撒币，导致勤劳却很贫穷，还被洗脑夸他们伟大

**Reply by KisaraKira** [reply]
又一个被洗脑洗残废的，到底是谁养谁啊？

**Reply by woshizengling** [reply]
> 印度也养活了14亿人。

拿印度说事，你也是个人才

**Reply by seeker-yao** [reply]
印度怎么了， 你有优越感了？

**Reply by woshizengling** [reply]
> 印度怎么了， 你有优越感了？

问印度怎么了？脑子真是个好东西，可惜你没有，别人批判还能说出个123，你只会口嗨

**Reply by seeker-yao** [reply]
> > 印度怎么了， 你有优越感了？
> 
> 问印度怎么了？脑子真是个好东西，可惜你没有，别人批判还能说出个123，你只会口嗨

你也没说出个123，你看你一副批判的语气，什么人才，脑子是个好东西，这啥东西。人多解决了温饱，就是制度优越？中国普通大众也不是没经历过灰暗时刻， 当年洋人嘲笑我们东亚病夫，也是你这种口气。

**Reply by woshizengling** [reply]
> 

你还知道中国普通大众经历过灰暗时刻哦，你还知道别人嘲笑过我们哦，看来历史没忘记啊，那中国是怎么站上今天这个位置的？是你们阴阳怪气换来的？还是你们喷出来的？别人开个repo的初心也是为了烂尾的业主能得到妥善解决，你们在这瞎逼逼制度

**Reply by seeker-yao** [reply]
> > 
> 
> 你还知道中国普通大众经历过灰暗时刻哦，你还知道别人嘲笑过我们哦，看来历史没忘记啊，那中国是怎么站上今天这个位置的？是你们阴阳怪气换来的？还是你们喷出来的？别人开个repo的初心也是为了烂尾的业主能得到妥善解决，你们在这瞎逼逼制度

钱没少给，房子没修， 还能是啥问题， 还能咋妥善解决，不就是各种讨论？让帖子火起来。我看这也帖子大家也没说啥过分的话，谈论制度怎么了，马哲毛概邓论还是必修课呢， 咱还得考试呢， 咋就不能说了。有些单位随意践踏人民的利益，还不能怀疑制度了？

**Reply by Liboce** [reply]
> > 你去看看互联网大会参加的都是什么国家😂
> 
> 
> 
> 你还知道中国普通大众经历过灰暗时刻哦，你还知道别人嘲笑过我们哦，看来历史没忘记啊，那中国是怎么站上今天这个位置的？是你们阴阳怪气换来的？还是你们喷出来的？别人开个repo的初心也是为了烂尾的业主能得到妥善解决，你们在这瞎逼逼制度

**Reply by John-zhan** [reply]
> 没有什么优势吧，不过资本主义也好不到哪里去，主要是看贫富差距哪种制度大，中国可以养活14亿人口充分说明制度的可行性

基本常识：纳税人养活了寄生虫，请对号入座各位

**Reply by mutumuda** [reply]
听我说谢谢你
因为有你 养活了傻逼

**Reply by DancerCoding** [reply]
共产党搞大跃进、文革批斗、计划经济、国家垄断土地（名义的土地改革实质的党地主，党成为最大的土地兼并者），最後是农民都被搞得快要饿死了才催生出土地承包责任制（实质是农名向党租借土地），最後才进行改革开放，如今大量劳动人民仍然要被剥削以养活上亿的党员干部，美其名曰特色社会主义，然后党大力宣传把这一切的功劳说成是政权和制度的优越性，这不是扯淡么

**Reply by stone0018** [reply]
> 共产党搞大跃进、文革批斗、计划经济、国家垄断土地（名义的土地改革实质的党地主，党成为最大的土地兼并者），最後是农民都被搞得快要饿死了才催生出土地承包责任制（实质是农名向党租借土地），最後才进行改革开放，如今大量劳动人民仍然要被剥削以养活上亿的党员干部，美其名曰特色社会主义，然后党大力宣传把这一切的功劳说成是政权和制度的优越性，这不是扯淡么

哦？那你来说怎么搞？历史唯物主义本身就是通过不断尝试进行解决问题，一个阶段的制度可以解决一个阶段的问题，同时也会存在相对的问题，既然你这么大的怨气，你来提出一个完美的制度来解决所有问题

**Reply by stone0018** [reply]
> > 共产党搞大跃进、文革批斗、计划经济、国家垄断土地（名义的土地改革实质的党地主，党成为最大的土地兼并者），最後是农民都被搞得快要饿死了才催生出土地承包责任制（实质是农名向党租借土地），最後才进行改革开放，如今大量劳动人民仍然要被剥削以养活上亿的党员干部，美其名曰特色社会主义，然后党大力宣传把这一切的功劳说成是政权和制度的优越性，这不是扯淡么
> 
> 哦？那你来说怎么搞？历史唯物主义本身就是通过不断尝试进行解决问题，一个阶段的制度可以解决一个阶段的问题，同时也会存在相对的问题，既然你这么大的怨气，你来提出一个完美的制度来解决所有问题

只是在这里进行抱怨，那你的解决思路是啥？
只在这里笔笔莱莱，u can u up

**Reply by DancerCoding** [reply]
> > > 共产党搞大跃进、文革批斗、计划经济、国家垄断土地（名义的土地改革实质的党地主，党成为最大的土地兼并者），最後是农民都被搞得快要饿死了才催生出土地承包责任制（实质是农名向党租借土地），最後才进行改革开放，如今大量劳动人民仍然要被剥削以养活上亿的党员干部，美其名曰特色社会主义，然后党大力宣传把这一切的功劳说成是政权和制度的优越性，这不是扯淡么
> > 
> > 
> > 哦？那你来说怎么搞？历史唯物主义本身就是通过不断尝试进行解决问题，一个阶段的制度可以解决一个阶段的问题，同时也会存在相对的问题，既然你这么大的怨气，你来提出一个完美的制度来解决所有问题
> 
> 只是在这里进行抱怨，那你的解决思路是啥？ 只在这里笔笔莱莱，u can u up

把独裁者的权力关在笼子里，推翻一党独裁，通过权力制衡和制度设计实现真正的依法治国。没有民主，一切都是空谈。建国後几乎所有规模化的反人类灾难，如大跃进饥荒、大锅饭运动、文革批斗惨案、汉字摧残简化、计划生育阉割汉族、计划经济共同贫困，以及目前的健康码电子镣铐随意赋红码禁足维稳、言论管控等等都是独裁政权带来的。

**Reply by John-zhan** [reply]
> > 印度怎么了， 你有优越感了？
> 
> 问印度怎么了？脑子真是个好东西，可惜你没有，别人批判还能说出个123，你只会口嗨

无脑喷的狱中人士

**Reply by ppinkiris** [reply]
> 印度也养活了14亿人。

你要不要先看看印度每年饿死多少人的情况下还在进行粮食出口？你说这话真不怕印度人骂你，有键盘真好

**Reply by percyjie** [reply]
> 印度也养活了14亿人。

龟龟 印吹都有市场

**Reply by blacklion-spec** [reply]
> 这就是被教育成功的典范，养活14亿人的是我们自己，我们劳动，我们交税，能来这的个税都少交不了，外加各种消费税。是我们养活了这些公务员，养活了这个政府，而不是他们养活我们，我们不买东西，不交税，他们瞬间就会急，就会用铁拳打人，如果这也叫养活的话，听我说，谢谢你。

是我们自己养活了自己

**Reply by blacklion-spec** [reply]
> > 印度也养活了14亿人。
> 
> 
> 
> 拿印度说事，你也是个人才

你也是个人才

**Reply by lzwbzd** [reply]
> 这就是被教育成功的典范，养活14亿人的是我们自己，我们劳动，我们交税，能来这的个税都少交不了，外加各种消费税。是我们养活了这些公务员，养活了这个政府，而不是他们养活我们，我们不买东西，不交税，他们瞬间就会急，就会用铁拳打人，如果这也叫养活的话，听我说，谢谢你。

印度十几亿畜牲给你点了个赞

**Reply by redapple2021** [reply]
> 

兄弟说的太透彻了。就是要监督权利，而不是某个人某个组织说了算

**Reply by marioabu** [reply]
> 
在我看来 政治制度和经济制度要分开说 资本主义并不等同于民主制度

**Comment by yaojun170** [comment]
所以我们叫特色社会主义

**Reply by yaojun170** [reply]
也就是雷锋和雷峰塔的关系

**Reply by xianyu403** [reply]
这个比喻绝了

**Reply by hclgdb** [reply]
哈哈哈哈

**Comment by shadowlgd** [comment]
有一点说错了，我们是专制制度，专制权力必然无限大，无限大的权力必然失衡，只有相互制约才能均衡

**Comment by JACWATA** [comment]
别问，问就是感恩

**Reply by white-cat** [reply]
这里不要谈论制度的问题！

**Reply by ababcabcd** [reply]
奴才当惯了，首先来个颅内监禁？

**Reply by heize** [reply]
颅内监禁，哈哈，词语先进性

**Reply by hclgdb** [reply]
应该是偷着乐 !!!

**Reply by tanuk1997** [reply]
> 奴才当惯了，首先来个颅内监禁？
良好的自我阉割能力

**Comment by macleer** [comment]
这个不好讨论

**Comment by MagicYangXX** [comment]
人家在维权 ，你们在这里瞎带jb节奏呢？纯纯的脑瘫么？

**Reply by zxcs001** [reply]
为什么不能讨论，做奴隶的自我修养？

我看到国内这帮韭菜们，在私下里说个话，还一副小心翼翼的那种模样。发自内心的让人感到恶心……

**Reply by yalo-ddaa** [reply]
讨论一个制度就是带节奏，哈哈

**Reply by alphagbone** [reply]
也就你这种sb多，奴性不灭

**Reply by miaodi** [reply]
奴才是这个样子的

**Reply by zongyezhou** [reply]
不解决根本问题，下次就是你。

**Reply by mutumuda** [reply]
老夫掐指一算，楼主年内必有铁拳临门，必被锤爆，此灾无药可解啊！

**Reply by ppinkiris** [reply]
这些人跟机器人一样，定体问，皿煮滋油听的好不叫人恶心，多少年没看到这种公知盛行的文体了，竟然有点熟悉，毕竟人家是工作，可以理解。

**Reply by blacklion-spec** [reply]
人民有权力，还需要在这里讨论吗？人民有权力还需要这样维权吗，这不是制度问题吗？这不是政治问题吗？政治和生活息息相关，你吃饭，你看电影，你买房，你存钱，你上班，你养老，那一件不是和政治息息相关？

**Reply by dofindme** [reply]
动不动就骂起来了。怪不得不能讨论，要封杀！

**Reply by Siryedu** [reply]
> 动不动就骂起来了。怪不得不能讨论，要封杀！

哈哈哈哈哈哈

**Comment by xiaohao587** [comment]
在五千年的960万平方公里土地上，从来就没有过民主，不过呢现在是这五千年中最糟糕的...，说真话给你抓起来，发帖子给你删掉，存银行钱让你取不出来，你闹就让白衣人和黑衣人打的头破血流，平时说的军民鱼水情，遇到事的时候换上一身衣服，把你打到卑服

**Reply by jiang443** [reply]
无论在哪里，自由永远都是有限度的

**Reply by woshizengling** [reply]
楼上那位，你试试去什么推特，脸书大放厥词看看，说得好像外面完全不管言论一样

**Reply by Legenries** [reply]
你...肯定不是很爱读书

**Reply by yalo-ddaa** [reply]
> 无论在哪里，自由永远都是有限度的

那您怎么看待权力的自由？
为什么权力机关想不让你取钱就可以不让你取钱，想赋红码就可以赋红码？
官老爷们的自由是有限度的吗？

**Reply by xiaer93** [reply]
> 
推特可以 评论总统、首相，国内微博很多字打不出来。

自由是有限度的，但是 自由的边界在哪里？你不觉得现在生活中处处是红线吗？

我们期望烂尾楼能够按时交付，只能在github这种网站交流，不是时代的悲哀吗？

**Reply by blacklion-spec** [reply]
> 无论在哪里，自由永远都是有限度的

依法治国，对人民来说，法无禁止即可为，对政府来说，法无授权不可为。

**Reply by alphagbone** [reply]
自由就是警察想打你就打你，想让你红码就红码，想封你就封你，而你只能听他说的话，这可不就是自由吗？

**Reply by John-zhan** [reply]
> 楼上那位，你试试去什么推特，脸书大放厥词看看，说得好像外面完全不管言论一样

感觉你俩这是在狱中受到有关部门指示在争取减刑，没有体会到世间疾苦

**Reply by pengjipan** [reply]
多学学历史，不会有坏处

**Reply by HumorChen** [reply]
去看看历史把，历史上轻描淡写的那些错误你再去国外看看比如维基百科之类的里面的纪实，你就明白其实我们处处被制约限制着，颅内监禁，为什么能悲哀到最后退守到github来，没点数？

**Reply by tanuk1997** [reply]
> 无论在哪里，自由永远都是有限度的

别人的边界线是你国的底裤吧

**Comment by hq001a** [comment]
大环境不好，哪种制度都有危机

**Reply by AnZhiYu** [reply]
你说了和没说有什么区别

**Reply by miaodi** [reply]
是的是的。

　　凡中国所有的，外国也都有。

　　外国人说中国多臭虫，但西洋也有臭虫；日本人笑中国人好弄文字，但日本人也一样的弄文字。不抵抗的有甘地；禁打外人的有希特拉；狄昆希吸鸦片；陀思妥夫斯基赌得发昏。斯惠夫德带枷，马克斯反动。林白大佐的儿子，就给绑匪绑去了。而裹脚和高跟鞋，相差也不见得有多么远。

　　只有外国人说我们不问公益，只知自利，爱金钱，却还是没法辩解。民国以来，有过许多总统和阔官了，下野之后，都是面团团的，或赋诗，或看戏，或念佛，吃著不尽，真也好像给批评者以证据。不料今天却被我发见了：外国也有的！

　　“十七日哈伐那电——避居加拿大之古巴前总统麦查度……在古巴之产业，计值八百万美元，凡能对渠担保收回此项财产者，无论何人，渠愿与以援助。又一消息，谓古巴政府已对麦及其旧僚属三十八人下逮捕令，并扣押渠等之财产，其数达二千五百万美元。……”

　　以三十八人之多，而财产一共只有这区区二千五百万美元，手段虽不能谓之高，但有些近乎发财却总是确凿的，这已足为我们的“上峰”雪耻。不过我还希望他们在外国买有地皮，在外国银行里另有存款，那么，我们和外人折冲樽俎的时候，就更加振振有辞了。假使世界上只有一家有臭虫，而遭别人指摘的时候，实在也不大舒服的，但捉起来却也真费事。况且北京有一种学说，说臭虫是捉不得的，越捉越多。即使捉尽了，又有什么价值呢，不过是一种消极的办法。最好还是希望别家也有臭虫，而竟发见了就更好。发见，这是积极的事业。哥仑布与爱迪生，也不过有了发见或发明而已。

　　与其劳心劳力，不如玩跳舞，喝咖啡。外国也有的，巴黎就有许多跳舞场和咖啡店。

　　即使连中国都不见了，也何必大惊小怪呢，君不闻迦勒底与马基顿乎？ ——

　　外国也有的！

十月十九日。

**Reply by hrsgn** [reply]
还是秦制最好了，希望再延续两千年，愿楼主来世还生种花家

**Comment by jiang443** [comment]
我相信对根本制度问题的讨论，不是志愿者做这个活动的目的

**Reply by zcaaron** [reply]
其实一切问题最后都会归根于根本制度

**Reply by xiaer93** [reply]
生活中，已经没有人谈论国事了。

**Reply by redapple2021** [reply]
因为现在你说就会抓你，胡温时期你说的最多河蟹掉，老江时期言论这块最松，那时候人心都很简单

**Reply by caofeibio** [reply]
1

**Comment by woshizengling** [comment]
别瞎带节奏好吧

**Reply by zxcs001** [reply]
为什么不要讨论，做奴隶的自我修养？

我看到国内这帮韭菜们，在私下里说个话，还一副小心翼翼的那种模样。发自内心的让人感到恶心……

**Reply by zxcs001** [reply]
> > 为什么不要讨论，做奴隶的自我修养？
> > 我看到国内这帮韭菜们，在私下里说个话，还一副小心翼翼的那种模样。发自内心的让人感到恶心……
> 
> 你是不是脑瘫啊，知道你在国外抓不着你。

还用个手机号来当账户，生怕别人不会搞社会工程学。来个短信爆破，或者人肉之类的。

注意点吧

**Comment by yalo-ddaa** [comment]
无论在哪里，自由永远都是有限度的

那您怎么看待权力的自由？
为什么权力机关想不让你取钱就可以不让你取钱，想赋红码就可以赋红码？
官老爷们的自由是有限度的吗？

**Reply by xiaer93** [reply]
刑不上士大夫，普通人改健康法刑法伺候，官员批量改健康码罚酒三杯，

这种灵活自由尺度，我实在不知道边界线在哪里。

**Comment by motion-dream** [comment]
> 通过最近的一系列问题，包括但不限于食品安全、村镇银行、停贷等，归根到底是怎样限制公权力。 大家还觉得社会主义制度有啥优越性吗？ 社会主义制度的意识形态是：公平正义、人类解放、共产主义、人民当家做主。 资本主义制度的意识形态是：自由民主、三权分立。

社会主义的优越性是有的。社会主义初级阶段，但是这个路吧已经分裂了。

**Reply by ray-hb** [reply]
我就感觉我们的社会主义初级阶段就是给社会主义反向带货，最后是自己人不信，外人更是笑而不语

**Comment by lucas-acc** [comment]
有事说事，不要宽泛地议论”制度“。

**Reply by Azheng197** [reply]
冰冻三尺，非一日之寒

**Reply by zongyezhou** [reply]
就是制度问题。

**Reply by blacklion-spec** [reply]
吃喝拉撒，从生到死，都和政治息息相关

**Reply by Pinruyi** [reply]
你就算就事论事，那导致断贷事件的原因是什么？难道不是期房预售制度？背后的监管制度没有问题？这一系列的制度难道不是把责任一股脑推给业主给购房者？又是因为什么，这个单方面责任的制度被设计出来，还被推行？这一切的一切环环相扣，是你想自甘当鸵鸟就能避开的？你又是为什么到了外面还不敢勇敢地讨论？

**Comment by 1510207073** [comment]
制度的优越性在于方便集中力量办大事，对于事情上有着统一的思想和认识，但也不可能否认，过于集中的权力会缺乏有效的纠正，缺少有效的监督，更可怕的在于如果集中了就很难再放下，让既得利益者放弃自己的利益是不可能的

**Reply by yjeff112** [reply]
所以很久以前一个伟人发动过一场自上由下的革命，在他走了后被全盘否定了

**Comment by crazynet999** [comment]
已经无可救药了，天朝

**Comment by liuwenwu520** [comment]
政府官员家庭资产透明化就能解决很多问题

**Reply by social-power** [reply]
还得先司法独立  否则他们糊弄老百姓 老百姓也拿他们没办法

**Reply by Novite5** [reply]
还可以洗钱、代持。

**Comment by AlphaXu917** [comment]
中央集权独断专行的特色资本主义

**Comment by clleo888** [comment]
社会主义制度的意识形态是：公平正义、人类解放、共产主义、人民当家做主。
资本主义制度的意识形态是：自由民主、三权分立。
**中国特色社会主义制度的意识形态是: 领导是上帝, 没有公平、全人类都定于一尊、财产全是党随时可用的、人民是家畜, 要多少繁殖,产出等等 都是党的资源。**

**Reply by clleo888** [reply]
有一個笑話: 中國出了一頭偉大的猪, 牠要把全中國都定於一猪, 厲害了!

**Reply by super3H** [reply]
你忘记改简体了，憨憨，假装大陆人失败

**Reply by markKscode** [reply]
> 你忘记改简体了，憨憨，假装大陆人失败

这和是台湾还是中国没有关系，人家说的都是实话。

**Reply by clleo888** [reply]
> 你忘记改简体了，憨憨，假装大陆人失败

对忘了转简体, 但是, 我转简体的目的是希望大家都看得懂, 不是想假装大陆人。你们什麽都以你骗我,我骗你的角度去看世界, 国家会好吗? 就是这原因你们很难明白外国人是怎样想的, 也不会明白外国的好生活条件是怎麽来的...

**Comment by WE-color** [comment]
烂尾楼说白了就是把最大的风险给了底层而已。真可怜。供着最高的房贷利率的房子，干着工作时间最久的工作，养着最幸福的官员，然后韭菜们还会主动替老爷们辩护，真是除了盐碱地全世界打灯笼都找不到这种玩意；

**Reply by wedxz** [reply]
哀其不幸,怒其不争

**Comment by smartleek** [comment]
你不会不知道他为什么加个特色吧

**Comment by Zoey-nn** [comment]
没有十全十美的制度，管他什么制度不制度，我觉得关键是权力的分配和制约 了，司法体系健全啊这些才是关键

**Reply by wedxz** [reply]
关键是你说的这些关键都是他们定的啊,你觉得他们会拿刀砍自己?

**Comment by yooq** [comment]
权利集中还能有什么优越性

**Reply by itsoursduty** [reply]
有是有的 比如疫情强制隔离控制，但是你指望这群蠢逼能合理的管控，简直天方夜谭 ，他们只会一刀切。

**Comment by super3H** [comment]
时间会说明一切，评判一个政府，一种制度，都要从其优缺点判断，没有完美的政府以及制度，当你否定他做的一切，只看到他的缺点，和瞎子没什么区别，同时当你只看到他的优点，看不到缺点也和瞎子没什么区别。辩证的看待，马哲都白学了？但凡有点脑子的都不会和这群憨憨起哄，国家垮台了，你会过得比现在难一辈子，还有台湾的憨憨，他们也失去战略价值，现在老美是不是过来摸摸你，还不是你有用。

**Reply by itsoursduty** [reply]
谁说一定会比现在难？感情现在比秦朝过的更苦咯？前面说的有点道理，然而当前这个制度的优势对于人民几乎等于没有。做不到无产阶级当家作主 还是赶紧下台吧 。别扯什么台湾，美国，别转移话题

**Reply by b4o** [reply]
叫有用吗？叫有用你可以当美国总统了，总有苍蝇，有用的方案一个没有，天天就知道乱叫狂吠，难道你觉得一定会比现在好？有什么根据？

**Comment by blacklion-spec** [comment]
> 人家在维权 ，你们在这里瞎带jb节奏呢？纯纯的脑瘫么？ 

人民有权力，还需要这么辛辛苦苦的维权，自断后路的维权？这不是政治问题？这不是制度问题？

**Comment by 3-ark** [comment]
就用这个烂尾来说下吧。但凡监管作为，而不是和开发商一起合力割韭菜，也不会这么多烂尾，所谓的制度优越性其实正是缺陷，短期看可以快速堆杠杆实现快速发展，但长期看因为杠杆加起来没限制，最终注定会崩溃，你还觉得这个是优势么？

**Comment by Siryedu** [comment]
你不关心政治，政治会关心你的！

**Comment by Siryedu** [comment]
你不关心政治，政治会关心你的！

**Comment by xijinping0** [comment]
> 社会主义制度的意识形态是：公平正义、人类解放、共产主义、人民当家做主。

楼主疑似外宾了。我国制度的优越性就是，红色江山代代传，十里山路不换肩。

人民当家做主？是赵家人当家作主！

[![不换肩](https://img.youtube.com/vi/cdH-zKCQs1M/0.jpg)](https://youtu.be/cdH-zKCQs1M?t=2)

https://github.com/xijinping0
《[习近平与他的情人们](https://lovers.xijinping.one)》

**Comment by 505Games-oss** [comment]
我看看我准备这个本本上有没有这个内容

**Comment by lettermessenger** [comment]
月满则亏，水满则溢。任何事物都是有期限的，一栋大楼建起来，这栋楼就已经注定了有坍塌的一天，时间问题而已

**Comment by gkrudtkfkd** [comment]
这个政府本身不是合法的，还谈什么制度？国家是由人民组成的，政府不是由人民选举产生的！也就是说政府不能代表人民！共党是非法霸占政府！

**Reply by b4o** [reply]
SB一个，以你的智商，能分清好坏吗？能看清自己未来几十年的人生吗？还淡一个国家的未来，给你这SB十个脑袋也选不对一个人

**Comment by ruo-xi** [comment]
> 通过最近的一系列问题，包括但不限于食品安全、村镇银行、停贷等，归根到底是怎样限制公权力。
> 大家还觉得社会主义制度有啥优越性吗？
> 社会主义制度的意识形态是：公平正义、人类解放、共产主义、人民当家做主。
> 资本主义制度的意识形态是：自由民主、三权分立。
> 

你觉得我们改开后还是社会主义？早就是个社会主义外壳了。

**Comment by Cegazhos** [comment]
这个主义我确实很失望，因为它还让台湾乡巴佬活着

**Comment by mxyulin** [comment]
油管去搜 翟山鹰-随心、随意、随笔
也就是地产党喊打喊杀的一个骗子老板（他确实承认骗了很多傻逼粉红的钱，利用傻逼粉红的一腔爱国心），翟山鹰父亲翟明战曾在空军军校最高任职副团、正团职参谋，空军大院红色系家庭。
他出国实质上是地产党要摘他公司多年来的财富和成果，结果和地产党没谈和，地产党想要杀人灭口，就听从挚友的劝诫跑到美国躲避政治迫害。
出于对中共地产党的恨意，他和王志安、郭文贵等反中共人士一样逐步披露中共地产党的种种奸邪淫恶。

**Reply by b4o** [reply]
这些蛀虫就应该派特工杀掉，在外面的中国人见到也应该杀掉，不管什么样的好制度，总有一些豺狼钻空子

**Reply by Cegazhos** [reply]
难绷，在AV种子网站里的那个郭文贵么wwwww

**Comment by b4o** [comment]
除了资本主义，社会主义，还有第三个吗？谁那么牛，来搞第三个出来 啊，，，，那些逢中比叫的狗的，也只是狗，不，连狗都不如

**Reply by oceanzhang860411** [reply]
懂什麼叫社會主義嗎？每個人都有基本的住房供給，醫療供給，教育供給，請問你有嗎？你買房是不是要掏空6個錢包？你看病的時候出多少比例？你孩子讀書是不是要搶學區房？你好意思說自己是社會主義嗎？連封建制度都不如的垃圾社會，還好意思說自己社會主義？

**Reply by DancerCoding** [reply]
> 除了资本主义，社会主义，还有第三个吗？谁那么牛，来搞第三个出来 啊，，，，那些逢中比叫的狗的，也只是狗，不，连狗都不如

共产党的红色食利阶级是最封建保守的，毕竟是建立在压榨劳动百姓基础上供养庞大的党政体制，但是你不要忘了还是要好好践行党的计划生育政策，结扎或者三胎可以灵活变换，要你做核酸就乖乖做，赋你红码就乖乖待在家里饿死最好，遇到迫害最好不要维权也不要喊冤，还有你的健康码行踪都被共产党监控和大数据利用，作为党的资源就要做到最好，做好为党献身的准备。

**Reply by Cegazhos** [reply]
> > 除了资本主义，社会主义，还有第三个吗？谁那么牛，来搞第三个出来 啊，，，，那些逢中比叫的狗的，也只是狗，不，连狗都不如
> 
> 共产党的红色食利阶级是最封建保守的，毕竟是建立在压榨劳动百姓基础上供养庞大的党政体制，但是你不要忘了还是要好好践行党的计划生育政策，结扎或者三胎可以灵活变换，要你做核酸就乖乖做，赋你红码就乖乖待在家里饿死最好，遇到迫害最好不要维权也不要喊冤，还有你的健康码行踪都被共产党监控和大数据利用，作为党的资源就要做到最好，做好为党献身的准备。

有脑子的都会说是中修了，也就你们台湾的傻逼还在那说红色。

**Comment by lance111l** [comment]
主义是被执政者或革命者利用的工具    关键是能够国富民强   
现在中国发展到世界第二   值得肯定     用你的方法不一定行
